services/drupal/config/sync/filter.format.filtered_html.yml

```diff
@@ -34,7 +34,7 @@ filters:
+      allowed_html: '<* accesskey class id role style tabindex title data data-* name hidden><a href hreflang occurrence id name target rel> <article> <div> <pre> <br> <p> <blockquote cite> <hr> <figure> <figcaption> <ol reversed start type> <ul type> <li> <dl> <dt> <dd> <address> <hgroup> <h2> <h3> <h4> <h5> <h6> <em> <i> <strong> <b> <small> <s> <section> <span> <sub> <sup> <mark> <abbr !title> <q cite> <cite> <ins data-date> <del cite datetime> <dfn> <time datetime pubdate> <var> <samp> <code> <kbd> <wbr> <bdo dir> <bdi> <ruby> <rt> <rp> <table summary width> <colgroup span> <col span> <caption> <tbody> <thead> <tfoot> <tr> <th colspan headers height rowspan scope width> <td colspan headers height rowspan scope width> <details open> <summary> <form action method> <fieldset disabled form name> <legend> <label for form> <input checked size type value disabled placeholder required readonly autocomplete autofocus> <textarea rows cols disabled placeholder required readonly autocomplete autofocus> <meter min max low high optimum form value> <select multiple> <optgroup disabled label> <option value> <output for form name> <button type value> <datalist> <keygen autofocus challenge disabled form keytype name> <progress max value> <img alt crossorigin height longdesc src width usemap data-caption data-align> <embed height src type width> <object form height name type usemap width> <param name value> <iframe allowfullscreen height name sandbox scrolling seamless src srcdoc width> <video autoplay buffered controls crossorigin height loop muted played preload poster src width> <audio autoplay buffered controls loop muted played preloadsrc> <source media src type> <track default kind label src srclang> <canvas height width> <map name> <area alt coords href hreflang media rel shape target type> <drupal-media data-caption data-align data-entity-type data-entity-uuid alt><drupal-inline-media data-entity-type data-entity-uuid>'
```

Comment:
@cherrypj Did you intentionally remove the aria-* attribute from all tags?

Reply:
Ohhhh. So that's why this work didn't land in live. I will fix.